app/command_autocomplete.go

```diff
@@ -198,7 +200,7 @@ func parseNamedArgument(arg *model.AutocompleteArg, parsed, toBeParsed string) (
 	return false, parsed + namedArg + " ", in[len(namedArg)+1:], model.AutocompleteSuggestion{}
 }
 
-func parseInputTextArgument(arg *model.AutocompleteArg, parsed, toBeParsed string) (found bool, alreadyParsed string, yetToBeParsed string, suggestion model.AutocompleteSuggestion) {
+func parseInputTextArgument(arg *model.AutocompleteArg, parsed, toBeParsed string, unquotedMultiWordAllowed bool) (found bool, alreadyParsed string, yetToBeParsed string, suggestion model.AutocompleteSuggestion) {
```

Comment:
naming: 1/5 `s/unquotedMultiWordAllowed/isLast` - more intention revealing, you can add a comment in-place that the last argument does not need to be quoted.

Reply:
Thanks for the feedback, I updated the variable name and pushed.